rapidpro/models/actions.py

```diff
@@ -53,12 +57,15 @@ def render(self):
 
 
 class SetContactFieldAction(Action):
-    def __init__(self, field_key, field_name, value):
+    def __init__(self, field_name, value):
         super().__init__('set_contact_field')
-        self.field_key = field_key
+        self.field_key = self._get_field_key(field_name)
         self.field_name = field_name
         self.value = value
 
+    def _get_field_key(self, field_name):
+        return field_name.strip().replace(' ', '_')
+
```

Comment:
Has this been checked against rapid pro? I thought capitalisation also mattered here?